Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -1010,7 +1005,7 @@ std::vector<const GameListItem*> CGameListCtrl::GetAllSelectedISOs() const
     item = GetNextItem(item, wxLIST_NEXT_ALL, wxLIST_STATE_SELECTED);
     if (item == wxNOT_FOUND)
       return result;
-    result.push_back(m_ISOFiles[GetItemData(item)]);
+    result.push_back(m_ISOFiles[GetItemData(item)].get());
```

Comment:
Can you also apply this to https://github.com/lioncash/dolphin/blob/f7ff1fb1ab15ee01ddd920c7ee73db6bc6f4647b/Source/Core/DolphinWX/GameListCtrl.cpp#L852 ?